src/components/leaderboard-screen/LeaderboardHighlight.tsx

```diff
@@ -0,0 +1,77 @@
+import { Medal, Trophy } from "lucide-react"
+
+import type { Doc } from "~/lib/db/types"
+import { cn } from "~/lib/utils"
+import UserAvatar from "../nav-bar/UserAvatar"
+import { Avatar } from "../ui/avatar"
+
+type LeaderboardTopThreeProps = {
+  players: [Doc<"users"> | undefined, Doc<"users"> | undefined, Doc<"users"> | undefined]
+}
+
+const PlayerCard = ({
+  player,
+  rank,
+  isGold,
+}: {
+  player: Doc<"users"> | undefined
+  rank: number
+  isGold?: boolean
```

Comment:
do we need both?

Reply:
Yeaah only cuz we showing their rank under their name but I think I can use `isGold` and `!isGold` in more places so I've made those changes and only using the rank to display their rank